Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -287,6 +287,29 @@ bool GameListItem::ReadPNGBanner(const std::string& path)
   return true;
 }
 
+const std::string GameListItem::GetFileType() const
+{
+  switch (GetBlobType())
+  {
+  case DiscIO::BlobType::PLAIN:
+    return wxTRANSLATE("Plain");
```

Comment:
Using `_()` in GameListCtrl.cpp is going to be problematic if we want to display the file type in more places later (like the game properties), but I don't see any problems with it other than that.